src/workers/blagoevgrad.ts

```diff
@@ -0,0 +1,224 @@
+/**
+ * Blagoevgrad adapter
+ * 
+ * @module blagoevgrads.ts
+ * @author vadiim <vadim123bg@gmail.com>
+ */
+
+// Import dependencies
+import BaseWorker from "./baseWorker";
+import puppeteer from "puppeteer";
+import { browserOptions } from "../config";
+import WorkerMessage from "../types/workerMessage";
+
+new class Blagoevgrad extends BaseWorker {
+
+    //Parse the date
+    private dataParser: { [key: string]: number } = {
```

Comment:
This should be normalized. We should be able to parse the month if we receive it in any case.